lib/puppet/provider/package/portage.rb

```diff
@@ -117,7 +117,7 @@ def qatom
       if match
         result_fields.zip(match.captures) do |field, value|
           # some fields can be empty or (null) (if we are not passed a category in the package name for instance)
-          if value == '(null)'
+          if value == '(null)' or '<unset>'
```

Comment:
I don't think this is doing quite what you expect here? This is the equivalent of `(value == '(null)') or true`. Did you mean `value == '(null)' || value == '<unset>'`?

Reply:
Can you tell I do more python than ruby? will fix